controller/persistence/api_session_store.go

```diff
@@ -49,13 +51,15 @@ func (entity *ApiSession) LoadValues(_ boltz.CrudStore, bucket *boltz.TypedBucke
 	entity.IdentityId = bucket.GetStringOrError(FieldApiSessionIdentity)
 	entity.Token = bucket.GetStringOrError(FieldApiSessionToken)
 	entity.ConfigTypes = bucket.GetStringList(FieldApiSessionConfigTypes)
+	entity.IPAddress = bucket.GetStringWithDefault(FieldApiSessionIPAddress, "")
```

Comment:
No migration needed. Post upgrade, any existing API Sessions will simply have blank IP addresses. New API Sessions will have the value populated.